### PR TITLE
Address Node version issue in acceptance test by changing tests to use latest official code samples and avoid ZIP test fixtures

### PR DESCRIPTION
**Pull request by SarahFrench**
Addressing this test failure ([100% since May 23rd 2024](https://hashicorp.teamcity.com/test/946081243120618630?currentProjectId=TerraformProviders_GoogleCloud_GOOGLE_NIGHTLYTESTS&expandTestHistoryChartSection=true)):

```
------- Stdout: -------
=== RUN   TestAccAppEngineFlexibleAppVersion_appEngineFlexibleAppVersionExample
=== PAUSE TestAccAppEngineFlexibleAppVersion_appEngineFlexibleAppVersionExample
=== CONT  TestAccAppEngineFlexibleAppVersion_appEngineFlexibleAppVersionExample
    vcr_utils.go:152: Step 1/2 error: Error running apply: exit status 1
        Error: Error creating FlexibleAppVersion: googleapi: Error 400: Error(s) encountered validating runtime. Your runtime version for nodejs is past End of Support. Please upgrade to the latest runtime version available..
          with google_app_engine_flexible_app_version.myapp_v1,
          on terraform_plugin_test.tf line 45, in resource "google_app_engine_flexible_app_version" "myapp_v1":
          45: resource "google_app_engine_flexible_app_version" "myapp_v1" {
--- FAIL: TestAccAppEngineFlexibleAppVersion_appEngineFlexibleAppVersionExample (176.31s)
FAIL
```

Zip file `mmv1/third_party/terraform/services/appengine/test-fixtures/hello-world.zip` defines the app engine service, contents:
Screenshot 2024-07-30 at 20 07 51

It's annoying to make updates to a zip file test fixture, so instead I've updated tests to create their own zip files. This means the test fixtures in this repo are easier to edit when bumping up runtime versions.

**Release Note Template for Downstream PRs (will be copied)**

```release-note:none

```

### CODE REVIEW

**Comment by SarahFrench** [comment]
TestAccAppEngineServiceNetworkSettings_appEngineServiceNetworkSettingsExample is having errors processing the zip file- I might have zipped by incorrect method

**Comment by SarahFrench** [comment]
TestAccAppEngineApplicationUrlDispatchRules_appEngineApplicationUrlDispatchRulesBasicExample

```
    vcr_utils.go:152: Step 1/2 error: Error running apply: exit status 1
        
        Error: Error waiting to create StandardAppVersion: Error waiting for Creating StandardAppVersion: Error code 3, message: Error processing zip file.
        
          with google_app_engine_standard_app_version.admin_v3,
          on terraform_plugin_test.tf line 16, in resource "google_app_engine_standard_app_version" "admin_v3":
          16: resource "google_app_engine_standard_app_version" "admin_v3" {
```

**Comment by SarahFrench** [comment]
In mmv1/third_party/terraform/services/appengine/resource_app_engine_standard_app_version_test.go I'm using sleeps to try navigate the permissions issue

**Comment by SarahFrench** [comment]
TestAccComputeRegionNetworkEndpointGroup_regionNetworkEndpointGroupAppengineExample should be addressed by 
e65ade2

File: mmv1/templates/terraform/examples/go/app_engine_flexible_app_version.tf.tmpl
Comment (SarahFrench) [inline mmv1/templates/terraform/examples/go/app_engine_flexible_app_version.tf.tmpl:1]: Need to fix this to being a zip

**Comment by SarahFrench** [comment]
I'm going to re-focus this PR to being about the test fixtures and remove the permissions stuff. That can go in a targeted PR

File: mmv1/templates/terraform/examples/go/app_engine_flexible_app_version.tf.tmpl
Comment (SarahFrench) [reply mmv1/templates/terraform/examples/go/app_engine_flexible_app_version.tf.tmpl:1]: Fixed in https://github.com/GoogleCloudPlatform/magic-modules/pull/11297/commits/55b0d5c8b094b5bb82c7a200ad9e21a8c9dfe331

**Comment by rileykarson** [comment]
Hey! I'm closing this PR as a part of a cleanup of older inactive PRs, using a threshold of PRs in "Draft" last updated over a month ago. This doesn't represent rejection of the change, and feel free to comment for me to reopen it if you plan to pick it back up, or feel free to start a new PR with the same changes in the future.

**Pull request closed without merging**